scrape.js

```diff
@@ -172,6 +175,19 @@ const scrapeLinkedIn = async (data) => {
 
       console.log(`Active users on page ${pageNumber}: `, activeEmployees);
 
+      if (!fs.existsSync("./output")) {
+        // check for existing output directory, create it if necessary
+        fs.mkdirSync("./output");
+      }
+
+      fs.appendFile(
+        fileName,
+        `Active users on page ${pageNumber}:  ${activeEmployees} \n`,
+        (err) => {
+          if (err) throw err;
+        }
+      );
+
```

Comment:
It would be great if it can be updated to store data in the following format (JSON):

```
{
    "activeProfiles": {
        "page0": [
            "link1",
            "link2"
        ],
        "page1": [
            "link1",
            "link2"
        ]
    }
 }

```